src/index.ts

```diff
@@ -104,6 +104,17 @@ export const createInstance = (config: Config): Instance => {
 
       return tx;
     },
+    isDomainMetadataLocked: async (
+      domainId: string,
+      signer: ethers.Signer
```

Comment:
Why are you taking a signer? There is a web3 provider from the config

Reply:
I checked before adding this and there isn't?

![image](https://user-images.githubusercontent.com/5976499/160679068-175f2be8-eb97-4711-8fa4-56a928ecf1e5.png)

Reply:
I was confused with zauction sdk and it having a provider. Please add a provider to the config so we follow the same patterns.